website/views/chatbot_api.py

```
@@ -0,0 +1,148 @@
+Answer STRICTLY about BLT features, hunts, leaderboards, bids, bacon tokens,
+projects, contributions, dashboards, GitHub syncing, and repository structure.
+
+User message: "{message}"
+"""
+
+    try:
+        completion = client.responses.create(
+            model="gpt-4o-mini",
+            input=prompt,
+        )
+        return completion.output_text
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)